.github/workflows/java-eks-retry.yml

```diff
@@ -0,0 +1,66 @@
+## Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+## SPDX-License-Identifier: Apache-2.0
+
+# This is a reusable workflow for running the Enablement test for App Signals.
+# It is meant to be called from another workflow.
+# Read more about reusable workflows: https://docs.github.com/en/actions/using-workflows/reusing-workflows#overview
+name: Java EKS Retry
+on:
+  workflow_call:
+    inputs:
+      aws-region:
+        required: true
+        type: string
+      test-cluster-name:
+        required: true
+        type: string
+      caller-workflow-name:
+        required: true
+        type: string
+
+concurrency:
+  group: 'java-eks-${{ inputs.aws-region }}-${{ github.ref_name }}'
```

Comment:
Motivation behind replacing `'${{ github.workflow }} @ ${{ inputs.aws-region }}'`? What does this value look like now?

We could also do something like `'${{inputs.test-cluster-name}} @ ${{ inputs.aws-region}}'`, because cluster is the limiting factor here.

Reply:
Mentioned the motivation in the PR description. 

I could add your suggestion for EKS cases, but your suggestion is difficult to implement in K8s case so we will still need `${{ github.ref_name }}` and `java-k8s`

Reply:
Works either way, understand the motivation now. Resolving comment